app/js/components/manageApps/AddonList.jsx

```diff
@@ -30,7 +30,15 @@ export const AddonList = ({ handleDownload, appList, openPage, openModal, update
                 />
               </td>
               <td onClick={() => openPage(app.appDetails)}>
-                <div className="addon-name">{app.appDetails && app.appDetails.name}</div>
+                <div className="addon-name">
+                  {
+                    app.appDetails.started == true || app.appDetails.started == undefined ?
```

Comment:
Using double equal too sign is not a sure way to check equality considering javascript's implicit coercion. Using a `===` is better. Also you can re-write this line with `{ app.appDetails && app.appDetails.started  ? pass condition : fail condition }`